wasm/jit/jit_arm64.go

```diff
@@ -476,20 +512,141 @@ func (c *arm64Compiler) compileLabel(o *wazeroir.OperationLabel) (skipThisLabel
 	return false
 }
 
+// compileUnreachable implements compiler.compileUnreachable for the arm64 architecture.
 func (c *arm64Compiler) compileUnreachable() error {
-	return fmt.Errorf("TODO: unsupported on arm64")
+	return c.exit(jitCallStatusCodeUnreachable)
 }
 
+// compileSwap implements compiler.compileSwap for the arm64 architecture.
 func (c *arm64Compiler) compileSwap(o *wazeroir.OperationSwap) error {
-	return fmt.Errorf("TODO: unsupported on arm64")
+	x := c.locationStack.peek()
+	y := c.locationStack.stack[int(c.locationStack.sp)-1-o.Depth]
```

Comment:
when there's multiple arithmetic maybe a trailing comment like `// Depth is relative to the last stack value` this makes it easier to think. just remember to update that comment if it ever changes :D